contracts/test-integration/BlsSigner.test.ts

```diff
@@ -567,6 +510,14 @@ describe("BlsSigner", () => {
       blsSigner,
     );
 
+    // BlsWalletWrapper.getRandomBlsPrivateKey from "estimateGas" method results in slightly different
+    // fee estimates. Which leads to a different signature. This fake avoids signature mismatch by stubbing a constant value.
+    sinon.replace(
```

Comment:
Example of differences observed without sinon.fake:


Test | Expected Fee Estimate | Fee Estimate
-- | -- | --
1 | 0.000079145258744232 | 0.000077749364772554
2 | 0.00006806648707833 | 0.000069299213638008
3 | 0.000060669559559918 | 0.000060666067046826

<br class="Apple-interchange-newline">